content/docs/guide-development.md

```diff
@@ -24,13 +24,26 @@ You can use a mock [lightcurve notebook](https://renkulab.io/gitlab/astronomy/mm
 ### Parametetrize the notebook 
 
 * create a cell with the following tag "parameters" (see [papermill manual](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#designate-parameters-for-a-cell)):
+  * if not annotated, the types of the inputs parameters are determined based on the parameter default value
+  * one can annotate the input parameter by putting comment with the `term` from the [ontology](https://odahub.io/docs/guide-ontology).
+  * several parameters are mandatory in the MMODA platform. These includes:
+    | Type annotation | Parameter default name |
```

Comment:
you can also explain that this parameter default name will be the name used to request the workflow
and for non-default - the provided name is used.
and the reasoning for having this mapping different depending on the parameter this (maybe in a note block or so)

Reply:
https://github.com/oda-hub/hugo-odahub/issues/13